pioneer/packages/joy-media/src/transport.ts

```diff
@@ -217,26 +227,27 @@ export abstract class MediaTransport extends Transport {
 
   async allPublicVideos (): Promise<VideoType[]> {
     const idOfPublicPS = (await this.allPublicationStatuses())
-      .find(x =>
+      .find((x) =>
         insensitiveEq(x.value, 'Public')
       )?.id;
 
     const idsOfCuratedCS = (await this.allCurationStatuses())
-      .filter(x =>
+      .filter((x) =>
         insensitiveEq(x.value, 'Under review') ||
         insensitiveEq(x.value, 'Removed')
-      ).map(x => x.id);
+      ).map((x) => x.id);
 
     const isPublicAndNotCurated = (video: VideoType) => {
       const isPublic = video.publicationStatus.id === idOfPublicPS;
```

Comment:
I populated the content directory with some content and I'm seeing this error:

![Screen Shot 2020-09-02 at 10 10 18 AM](https://user-images.githubusercontent.com/1621012/91956584-e9c02000-ed15-11ea-8b42-3c6d71cd279a.png)

Reply:
For context, while exporting state from current chain, some items were dropped and their property values were dropped. This could explain this error for some particular videos: https://github.com/Joystream/joystream-api-examples/pull/12/files#diff-6d0009198c779316ce8a37be5c378e3cR118

Reply:
Perhaps the media transport needs some improvement to determine when the property values don't exist and just exclude it from the array (similarly to where we did this check `entity.in_class_schema_indexes.toArray().length`)